src/rust/bitbox02-rust/src/async_usb.rs

```diff
@@ -13,11 +13,11 @@
 // limitations under the License.
 
 //! This module provides the executor for tasks that are spawned with an API request and deliver a
-//! USB response.
+//! USB response. Terminology: host = computer, client = BitBox02.
```

Comment:
Nice! But how about "device" or "peripheral"? Client sounds ambiguous imho and typical reminology in USB docs I've seen are hosts, hubs, peripherals and devices.

Reply:
Done